relay-event-normalization/src/clock_drift.rs

```diff
@@ -164,7 +164,9 @@ mod tests {
             contexts: {
                 let mut contexts = Contexts::new();
                 contexts.add(TraceContext {
-                    trace_id: Annotated::new(TraceId("4c79f60c11214eb38604f4ae0781bfb2".into())),
+                    trace_id: Annotated::new(
+                        TraceId::parse_str("3c79f60c11214eb38604f4ae0781bfb2").unwrap(),
```

Comment:
```suggestion
                       "3c79f60c11214eb38604f4ae0781bfb2".parse().unwrap(),
```

Since it implements `FromStr`

Reply:
Is there a particular reason to change the ID itself here?